src/web/src/config.ts

```diff
@@ -15,6 +15,9 @@ const searchServiceUrl = process.env.NEXT_PUBLIC_SEARCH_URL;
 const feedDiscoveryServiceUrl = process.env.NEXT_PUBLIC_FEED_DISCOVERY_URL;
 const statusUrl = process.env.NEXT_PUBLIC_STATUS_URL;
 
+const supabaseUrl = process.env.SUPABASE_URL;
```

Comment:
For these to work, we have to forward them as build `ARGS` into the docker build process.  You need to add them to:

- https://github.com/Seneca-CDOT/telescope/blob/master/docker/docker-compose.yml#L276-L286
- https://github.com/Seneca-CDOT/telescope/blob/master/src/web/Dockerfile

Follow what we do with the variables above to see the pattern.

Reply:
Done, let me know if I followed the convention correctly